data/Audiences.ts

```diff
@@ -1,23 +1,33 @@
 import { Audience } from "../types/targeting";
 
-const audiences :  {[id: string] : Audience } = {
+const CHANNEL_NIGHTLY = "Nightly"
```

Comment:
I'd skip these constants in favor of making `firefox_channel`'s type a enum, which provides similar guarantees.